### PR TITLE
Fixing windows issue, also added 'build/' to .eslintrc

### PR DESCRIPTION
**Pull request by ajmchambers**
Had a go at getting it working on windows:
- added library `cross-env` to
get the `NODE_PATH=source`'s working on windows
- fixed a couple of path's in the webpack config's

Unrelated but noticed lint was trying to lint the 'build/' folder (when
it existed) and failing (on both windows and mac) so added it to the
.eslintignore

_(I'm still learning github and this whole process in general so let me know if I'm doing anything wrong, cheers.)_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
     "build": "npm run clean && npm run build:webpack",
-    "start": "NODE_PATH=source babel-node devServer.js",
-    "server": "NODE_PATH=source babel-node source/server/index.js",
-    "test": "NODE_PATH=source babel-node source/test/index.js",
+    "start": "cross-env NODE_PATH=source babel-node devServer.js",
+    "server": "cross-env NODE_PATH=source babel-node source/server/index.js",
+    "test": "cross-env NODE_PATH=source babel-node source/test/index.js",
     "lint": "eslint --cache .",
-    "watch": "watch 'clear && npm run lint -s && npm run test -s' source",
+    "watch": "watch \"clear && npm run lint -s && npm run test -s\" source",
```
Comment (ericelliott) [inline package.json]: Was this needed to get it working on Windows?

**Comment by ericelliott** [comment]
Looks good. Thanks!

File: package.json (hunk 1 above)
Comment (nkbt) [inline package.json]: Yep. Windows does not allow single quotes.

**Comment by nkbt** [comment]
Good one :+1:

**Comment by gacosta89** [comment]
nice

File: package.json (hunk 1 above)
Comment (ericelliott) [inline package.json]: Stupid Windows. ಠ_ಠ

**Pull request merged**